functions.py

```diff
@@ -146,7 +146,7 @@ def get_jenkins_job_info(server, job_name):
 
 		lcb_url = build_info['url']
 		lcb_result = build_info['result']
-		compose = [action['text'][13:-4] for action in build_actions if 'core_puddle' in action.get('text', '')]
+		compose = [action['html'][-42:-13] for action in build_actions if 'core_puddle' in action.get('html', '')]
```

Comment:
```suggestion
		compose = [str(action['html']).split('core_puddle:')[1].split('<')[0].strip() for action in build_actions if 'core_puddle' in action.get('html', '')]
```
Its not too pretty but saves going through the whole parsing routine with HTMLParser or Beautifulsoup, so this should work for now

Reply:
Tested it on all supported versions and got the desired ouput, in previous patch the versions 13 and 10 were not showing proper results, Example: ck">core_puddle: 2020-08-05.1 for one of the OSP 13 jobs

Reply:
Ack, will update the patch with this line